### DISCUSSION TITLE
DataGrid : How to show last page when a row is inserted ?

### DISCUSSION THREAD

**Discussion by cbs-cbt** [Megabit/Blazorise#1687] [Q&A]
To show the last inserted item, I'd like to display the last page of the grid when the "OnRowInserted" event is generated. To do this, I was thinking of using the CurrentPage property, but the documentation contraindicates using it from outside the element. Do you know how I can achieve this?

**Comment by stsrki** [comment]
Nothing stops you from using `CurrentPage`. It's marked with `[Parameter]` attribute so it's perfectly safe.

**Reply by cbs-cbt** [reply]
Thank you Mladen for the very fast answer.
I will then use `CurrentPage`.
